provision/minikube/allinone.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
can we rename this shell script to `bootstrap.sh` or `install-and-config-services.sh` ?

Reply:
Yes, eventually I might settle for something like `rebuild.sh`, as the first step is to destroy an old installation. Let's wait and see until after the demo.

Reply:
I missed this comment, tentatively proposing to move to a `Makefile`: https://github.com/ahus1/keycloak-benchmark/pull/1

Reply:
I've renamed the script to `rebuild.sh`. There's now also `upgrade.sh` to install the latest changes.